### PR TITLE
Update forwarding service to work with ad server

### PR DESCRIPTION
**Pull request by msub2**
Notable differences from SDK:
- Does not include helpers file, as we cannot check user platform with the forwarding service

### CODE REVIEW

File: utils/networking.js [hunk 1]
```diff
@@ -1,175 +1,74 @@
-const axios = require('axios');
-const formatImport = require('../utils/formats.js');
-const helpers = require('../utils/helpers.js');
-const BEACON_GRAPHQL_URI = 'https://beacon2.zesty.market/zgraphql';
-const BEACON_GRAPHQL_GENERAL_URI = 'https://beacon2.zesty.market/graphql';
+import axios from 'axios';
+import { formats } from '../utils/formats.js';
+//import { v4 as uuidv4 } from 'uuid'
 
-const ENDPOINTS = {
```
Comment (limbofeather) [inline utils/networking.js:7]: Thanks for getting around to removing these

File: utils/networking.js [hunk 2]
```diff
@@ -1,175 +1,74 @@
...
-    if (campaign) {
-      const campaignId = sellerAuctions[0].buyerCampaignsIdList[i]; // Graph stores as string, coerce to int
-      latestAuction = sellerAuctions[0].buyerCampaigns.find(campaign => campaign.id === campaignId)
-    }
-  });
-  
-  if (latestAuction == null) {
-    return DEFAULT_DATAS 
-  }
+const DB_ENDPOINT = 'https://api.zesty.market/api';
```
Comment (limbofeather) [inline utils/networking.js]: These aren't necessarily the DB, but the servers correct?

File: utils/networking.js [hunk 3]
```diff
@@ -1,175 +1,74 @@
...
+// TODO: Determine best way to enable switching to staging
```
Comment (limbofeather) [inline utils/networking.js]: I would recommend using a flag that gets toggled through an environment variable on the machine.

File: utils/formats.js [hunk 4]
```diff
@@ -1,40 +1,36 @@
-const helpers = require('./helpers');
+const assetsURL = 'https://zesty-storage-prod.s3.amazonaws.com/images/zesty';
```
Comment (limbofeather) [inline utils/formats.js:1]: We should probably put this endpoint behind AWS CloudFront/Cloudflare to enable caching

File: README.md [hunk 5]
```diff
@@ -1,127 +1,107 @@
 # Forwarding Service
```
Comment (limbofeather) [inline README.md:1]: Let's link to this in our docs too!

**Review by limbofeather — APPROVED** [review]
Thanks @msub2 - LGTM 🍏

File: README.md (hunk 5 above)
Comment (msub2) [reply README.md:1]: Done

File: utils/networking.js (hunk 3 above)
Comment (msub2) [reply utils/networking.js]: Will handle enabling this for forwarding+SDK in the future

File: utils/formats.js (hunk 4 above)
Comment (msub2) [reply utils/formats.js:1]: Can definitely look into this yeah

File: utils/networking.js (hunk 2 above)
Comment (msub2) [reply utils/networking.js]: Correct, adjusted var wording

**Pull request merged**